### ISSUE TITLE
Homebridge Broadlink RM Pro might not be ready for Homebridge v2.

### ISSUE THREAD

**Issue by JasonGoldenDDT** [kiwi-cam/homebridge-broadlink-rm#730]
Is there a time estimate for this plug in to become 2.0 compliant?  What support if any can the community or users provide to assist in this remediation?

<img width="790" alt="Screenshot 2024-08-29 at 7 25 41 AM" src="https://github.com/user-attachments/assets/200a35e5-3d57-48ab-be26-8cecd6e6a1fa">

**Comment by maxileith** [comment]
It is already homebridge 2.0 compliant

**Comment by JasonGoldenDDT** [comment]
Homebridge is still flagging latest version.

<img width="579" alt="Screenshot 2024-09-23 at 9 03 28 AM" src="https://github.com/user-attachments/assets/8599e5d9-4793-46b1-a6b6-41f2568d782a">

**Comment by maxileith** [comment]
Yes, because the plugin dev has not specifically marked the plugin as compatible. But that does not mean it is not compatible. Two different things.

Release notes show that it is compatible. Besides that, the plugin does not use the methods that have been removed from v2.

**Comment by JasonGoldenDDT** [comment]
Sure.  As an end user I'm reacting to the information displayed by homebridge.  Perhaps they need to be flagged on the status of the plug in so your end users aren't given inaccurate information.

**Comment by maxileith** [comment]
> your end users

I want to make clear that I am not the plugin developer.

**Comment by maxileith** [comment]
Not stale

**Comment by JasonGoldenDDT** [comment]
can someone summarize the roadblock here?  do we need to back out of this plugin and migrate to a different solution?

**Comment by kiwi-cam** [comment]
The only roadblock is that homebridge 2.0 is in Beta. To clear this warning, I need to mark the live release of the plugin as compatible with the beta version of homebridge 2.0 - which is bad practice, and I don't want to do it.

If you install the beta version of this plugin, you'll see that it is marked compatible with homebridge 2.0:

![Image](https://github.com/user-attachments/assets/64eb7eb9-8754-46a8-8d96-808d2d69a795)
